admin/devurls.md

```diff
@@ -51,3 +51,45 @@ The final step to enabling dev URLs is to update your wildcard DNS record. Get
 the **ingress IP address** using `kubectl --namespace coder get ingress` and
 point your wildcard DNS record (e.g., \*.my-custom-domain.io) to the ingress IP
 address.
+
+## Setting dev URL access permissions
+
+Once you've enabled dev URLs for users, you can set the **maximum access
+level**. To do so, go to **Manage** > **Admin**. On the **Infrastructure** tab,
+scroll down to **Dev URL Access Permissions**.
+
+<table>
+  <tr>
+    <th>Maximum access level</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <th>Public</th>
+    <td>Accessible by anyone (no authentication required)</td>
```

Comment:
```suggestion
    <td>Accessible by anyone (no authentication required) with access to the network your cluster is on</td>
```

Public is a scary term for people and I've been mitigating it by saying "public to your network" on demo calls. This is suggested wording, please word smith it as you see fit 😋